apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameter.cs

```diff
@@ -34,6 +34,7 @@ public sealed class SpannerParameter : DbParameter, ICloneable
         private SpannerDbType _spannerDbType;
 
         /// <inheritdoc />
+        // Using DbType of VarNumeric will always default to SpannerDbType of SpannerNumeric and never to PgNumeric.
```

Comment:
So that this makes it into public docs, I think you have to stop inheriting the docs, rewrite them here, and add this on the Remarks section.